File: 01 Python Programming Fundamentals/Lesson 3 Conditionals/Conditionals.py
#!/bin/python3
''' 
Created by Jonatan Gomez Garcua on 04-01-2024
'''

try:
    userData = int(input('Input the number 2: '))
    
    if userData == 2:
        #First condition is true
        print('Thanks for input the number 2')
    elif userData < 0:
        #Second condition is true
        print('You have been inputted a negative number')
    else:
        #If both conditions are false
        print('Case not supported by condition')
except:
    print('The value entered is not an integer value')

File: 00 Python Programming Fundamentals/Lesson 3 Conditionals/Conditionals.cpp
#include <iostream>
using namespace std;
/*
Created by Jonatan Gomez Garcia on 04-01-2024
*/


int main() {
int userData;

cout << "Input the number 2: ";
cin >> userData;


   if(userData == 2){
    //First condition is true
    cout << "Thanks for input the number 2" << endl;
   } else if (userData < 0)
    //Second condition is true
    cout << "You have been inputted a negative number" << endl;
   else {
    //If both condition are false
    cout << "Case not supported by condition" << endl;
   }

    return 0;
}